### DISCUSSION TITLE
Vercel stuff

### DISCUSSION THREAD

**Discussion by awsome-webdev** [3kh0/ext-remover#1277] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
these are the vercel websites i found that have ai (all are free and working as of now), or are prolly going to be handy at some point for cheating, some are games, and other cool apps

google dork for finding these: ""  inurl:"vercel.app" site:"vercel.app"
> this just show all the vercel deployments out their and i just looked through them, i got to page *33* on google

- [2txt](https://2txt.vercel.app/)
> image to text
- [cebras (ai)](https://cerebras.vercel.app/)
> ai assistant, not text completely voice
- [khanswers](https://khanswers.vercel.app/)
>does your kahoot for you
- [whybot](https://whybot-khaki.vercel.app/)
> a ai researcher that makes a graph of questions and answers (coolest one in my opinion)
- [Hurdle](https://playhurdle.vercel.app/)
> a harder version of wordle
- [fast type](https://fast-type-zeta.vercel.app/)
>a not ad infested way to improve typing
- [Murdle](https://murdle.vercel.app/)
>hang man death with wordle
- [resent client](https://resent4-0.vercel.app/)
>i know that resent client is a popular eagler craft client but i didn't know if this was the official link or and alt
- [wingpt](https://wingpt.vercel.app/)
> a chatgpt client on windows 95 for free 
- [hello Llama](https://hello-llama.vercel.app/)
> llama chat
- [TrackerHub](https://trackerhub.vercel.app/)
> didn't test but plays music in google sheets, thought might be useful people with music blocked

**Comment by akabutnicer** [comment]
actually a somewhat useful and rational post. I'm proud.

**Reply by awsome-webdev** [reply]
thnx ig,
i just realized you can pit the cerebras ais against each other if you have two devices 😂

**Reply by akabutnicer** [reply]
ah, but i guess that also goes for any artificial AI voice site/app

**Reply by awsome-webdev** [reply]
ya

**Comment by akabutnicer** [comment]
also uh shameless plug, but take some time out your day to look at the discussion I made 🙏
